app/va/va_profile/va_profile_client.py

```diff
@@ -175,58 +292,62 @@ def get_is_communication_allowed(
 
-    def _make_request(
-        self,
-        url: str,
-        va_profile_id: str,
-        bio_type: str = None,
-    ):
-        start_time = monotonic()
-
-        self.logger.info('Querying VA Profile with ID %s', va_profile_id)
-
-        try:
-            response = requests.get(url, cert=(self.ssl_cert_path, self.ssl_key_path), timeout=(3.05, 1))
-            response.raise_for_status()
-
-        except requests.HTTPError as e:
+    def _handle_exceptions(self, va_profile_id, error):
+        if isinstance(error, requests.HTTPError):
             self.logger.warning('HTTPError raised making request to VA Profile for VA Profile ID: %s', va_profile_id)
-            self.statsd_client.incr(f'clients.va-profile.error.{e.response.status_code}')
+            self.statsd_client.incr(f'clients.va-profile.error.{error.response.status_code}')
 
             failure_reason = (
-                f'Received {responses[e.response.status_code]} HTTP error ({e.response.status_code}) while making a '
+                f'Received {responses[error.response.status_code]} HTTP error ({error.response.status_code}) while making a '
                 'request to obtain info from VA Profile'
             )
 
-            if e.response.status_code in [429, 500, 502, 503, 504]:
-                exception = VAProfileRetryableException(str(e))
+            if error.response.status_code in [429, 500, 502, 503, 504]:
+                exception = VAProfileRetryableException(str(error))
                 exception.failure_reason = failure_reason
 
-                raise exception from e
-            elif e.response.status_code == 404:
-                exception = VAProfileIDNotFoundException(str(e))
+                raise exception from error
+            elif error.response.status_code == 404:
+                exception = VAProfileIDNotFoundException(str(error))
 
-                raise exception from e
+                raise exception from error
             else:
-                exception = VAProfileNonRetryableException(str(e))
+                exception = VAProfileNonRetryableException(str(error))
                 exception.failure_reason = failure_reason
 
-                raise exception from e
+                raise exception from error
 
-        except requests.RequestException as e:
+        elif isinstance(error, requests.RequestException):
             self.statsd_client.incr('clients.va-profile.error.request_exception')
 
             failure_message = 'VA Profile returned RequestException while querying for VA Profile ID'
 
             exception = VAProfileRetryableException(failure_message)
             exception.failure_reason = failure_message
 
-            raise exception from e
+            raise exception from error
 
-        except requests.Timeout:
+        elif isinstance(error, requests.Timeout):
             self.logger.error('The request to VA Profile timed out for VA Profile ID %s.', va_profile_id)
             raise
```

Comment:
I believe we need to move `Timeout` above the more general `RequestsException` if we want to see this logging properly.